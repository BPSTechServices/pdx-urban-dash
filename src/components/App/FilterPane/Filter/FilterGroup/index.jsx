import React from "react";
import { Container } from "reactstrap";
import PropTypes from "prop-types";

export default class FilterGroup extends React.Component {
  static propTypes = {
    title: PropTypes.string.isRequired
  };

  constructor(props) {
    super(props);
    this.title = "";
  }

  render() {
    return (
      <Container style={{ borderLeft: "2px solid rgba(0,0,0,.1)" }}>
        <h2 style={{ marginBottom: "1rem" }}>{this.props.title}</h2>
        {this.props.children}
      </Container>
    );
  }
}
