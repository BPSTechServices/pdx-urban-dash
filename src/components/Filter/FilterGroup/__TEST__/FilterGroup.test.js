import React from 'react';
import { shallow } from 'enzyme';
import FilterGroup from '../index';

describe('FilterGroup Render', () => {
  test('snapshot', () => {
    const component = shallow(<FilterGroup title="test"><p>test</p></FilterGroup>);
    expect(component).toMatchSnapshot();
  });
});
