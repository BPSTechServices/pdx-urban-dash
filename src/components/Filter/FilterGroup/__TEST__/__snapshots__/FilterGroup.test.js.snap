// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilterGroup Render snapshot 1`] = `ShallowWrapper {}`;
